src/event.rs

```diff
@@ -70,7 +70,7 @@ impl<T> Event<T> {
 }
 
 /// The reason the event loop is resuming.
-#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
```

Comment:
Why remove 'Hash' here?

Reply:
There is no implementation of the Hash Trait in `instant` library. And I saw it was removed on master https://github.com/rust-windowing/winit/blob/master/src/event.rs#L156

Reply:
Hm, seems like we might need to merge master into gamepad-device-events. Either way, makes sense 👍